src/NetscriptFunctions/Infiltration.ts

```diff
@@ -81,5 +129,145 @@ export function NetscriptInfiltration(): InternalAPI<NetscriptInfiltation> {
       const locationName = getEnumHelper("LocationName").nsGetMember(ctx, _locationName);
       return calculateInfiltrationData(ctx, locationName);
     },
+    getMarketDemand: (ctx) => () => {
+      checkAccess(ctx);
+      return calculateMarketDemandMultiplier(Date.now(), false);
+    },
+    startInfiltration: (ctx) => (_locationName) => {
+      checkAccess(ctx, 3);
+      const locationName = getEnumHelper("LocationName").nsGetMember(ctx, _locationName);
```

Comment:
It would be nice to have a default like `_locationName ?? Player.location` for simplicity